CHANGELOG.md

```diff
@@ -1,13 +1,34 @@
 Change Log
 ===
-<!--
-v4.0.0-dev.2 (not released)
+
+v4.0.0
 ---
 * **BREAKING CHANGES**
+    * [api-change] Minor changes to the API. See PR for changes and upgrade instructions (#1720)
     * [removed-rule] Removed `no-unreachable` rule; covered by compiler (#661)
-* [enhancement] `--fix` option added to automatically fix selected rules (#1697)
+    * [enhancement] Changed order of applied configuration files for the `extends` array (#1503)
```

Comment:
explain "why", not just "what". in this case, this change makes makes the configuration field more intuitive